src/Sentry.Unity.Editor/Native/SentryWindowsPlayer.cs

```diff
@@ -0,0 +1,140 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using Sentry.Extensibility;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Sentry.Unity.Editor.Native
+{
+    internal static class SentryWindowsPlayer
```

Comment:
This class has a lot of methods taking `IDiagnosticLogger`. It seems to me this should be an instance class instead of static.